BUILD.bazel

```diff
@@ -8,6 +8,26 @@ load("@//bazel:cython_library.bzl", "pyx_library")
 
 COPTS = ["-DRAY_USE_GLOG"]
 
+# grpc common lib
+cc_library(
+    name = "common_grpc_lib",
+    srcs = [
+        "src/ray/rpc/grpc_server.cc",
+    ],
+    hdrs = [
+        "src/ray/rpc/client_call.h",
+        "src/ray/rpc/grpc_server.h",
+        "src/ray/rpc/server_call.h",
+        "src/ray/rpc/util.h",
```

Comment:
I think we keep the common rpc files under `rpc` folder, and move `node_manager/object_manager` rpc files to `rpc/node_manager` and `rpc/object_manager`. then you don't need to list all the files here. You can simply use `glob`.

Reply:
+1 for moving these files